app/com/lynxanalytics/biggraph/frontend_operations/ExportOperations.scala

```diff
@@ -153,4 +154,96 @@ class ExportOperations(env: SparkFreeEnvironment) extends OperationRegistry {
         .map(_ => wc.createSnapshotFromState(user, params("path"), getState))
     }
   })
+
+  abstract class Neo4jAttributeExport(context: Operation.Context) extends TriggerableOperation(context) {
+    params ++= List(
+      Param("url", "Neo4j connection", defaultValue = "bolt://localhost:7687"),
+      Param("username", "Neo4j username", defaultValue = "neo4j"),
+      Param("password", "Neo4j password", defaultValue = "neo4j"),
+      NonNegInt("version", "Snapshot revision ID", default = 1))
+    lazy val project = projectInput("graph")
+    val nodesOrRelationships: String
+    override def enabled = FEStatus.enabled
+    override def trigger(wc: WorkspaceController, gdc: GraphDrawingController) = {
+      gdc.getComputeBoxResult(List(exportResult.gUID))
+    }
+    override def getOutputs(): Map[BoxOutput, BoxOutputState] = {
+      params.validate()
+      Map(context.box.output(
+        context.meta.outputs(0)) -> BoxOutputState.from(exportResult, params.toMap - "password"))
+    }
+    def exportResult() = {
+      val keys = splitParam("keys")
+      val attrs = (splitParam("to_export") ++ keys).toSet.toList
+      val t = graph_operations.AttributesToTable.run(
+        attrs.map(a => a -> project.vertexAttributes(a)))
+      assert(keys.nonEmpty, "You have to choose one or more attributes to use as the keys for identifying the nodes in Neo4j.")
+      val op = graph_operations.ExportAttributesToNeo4j(
+        params("url"), params("username"), params("password"), params("labels"), keys,
+        params("version").toInt, nodesOrRelationships)
+      op(op.t, t).result.exportResult
+    }
+  }
+
+  registerOp(
+    "Export vertex attributes to Neo4j", defaultIcon, ExportOperations,
+    List("graph"), List("exported"), new Neo4jAttributeExport(_) {
+      import Operation.Implicits._
+      params ++= List(
+        Param("labels", "Node labels", defaultValue = ""),
+        Choice("keys", "Attribute to use as key",
```

Comment:
```suggestion
        Choice("keys", "Attributes to use as key",
```

Reply:
I've intentionally put it in the singular. I think 99% of the cases you have a single key. You need to somehow identify nodes, right? If you use first name + last name + birth date you will have issues.

But it's fairly clearly a multiple-choice parameter and it's documented in the help. (Not yet committed.)

Reply:
Hmm, the fact that the name and the kind conflicts is exactly what I would find strange/confusing and I'm not sure this is a good way to encourage people to use a single key attribute.  What about attribute(s) as a compromise? :)

Reply:
> What about attribute(s) as a compromise? :)

Done.